packages/suite/src/actions/suite/metadataLabelingActions.ts

```diff
@@ -25,21 +32,44 @@ import type { MetadataAction } from './metadataActions';
 import * as metadataActions from './metadataActions';
 import * as metadataProviderActions from './metadataProviderActions';
 
+type LabelableEntity = ReturnType<ReturnType<typeof getLabelableEntities>>[number];
```

Comment:
As we discussed on the call, what is called `LabelableEntity` now is only the Device and an Account. However we also add labels to transactions. 

I suggest something like "EntityThatHoldsLabelData" (ofc not literally :D) We need to find more suitable name along those lines.